flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/result/CollectStreamResult.java

```diff
@@ -52,12 +52,18 @@
 	private final ResultRetrievalThread retrievalThread;
 	private final JobMonitoringThread monitoringThread;
 	private ProgramDeployer<C> deployer;
+	private final ClassLoader classLoader;
```

Comment:
Nit: I would move it one line up to distinguish the `final` fields from the "runtime" ones.

Reply:
👌